benchexec/resources.py

```diff
@@ -15,191 +10,157 @@
 import math
 import os
 import sys
+from functools import cmp_to_key
 
-from benchexec import cgroups
-from benchexec import util
+import cgroups
+import util
```

Comment:
Please use the original way the imports are written and not relative imports. For example, right now the tests in CI cannot be executed due to this.